daemons/dss-notify-v2/Readme.md

```diff
@@ -80,6 +80,10 @@ The bundle metadata document format for a new bundle or version is
 ```
 {
   "event_type": "CREATE",
```

Comment:
Do you have to specify an event_type in order to get into bundle info?Would it be possible to not specify any event_type and get **any** event_type for some bundle_uuid?

Reply:
I think you're asking about JMESPath filtering. Yes, it would be possible to filter on bundle_info.uuid without filtering on event_type.